AIMSICD/src/main/java/com/secupwn/aimsicd/utils/RequestTask.java

```diff
@@ -394,4 +437,15 @@ private void showHideMapProgressBar(boolean pFlag) {
     public static String getOCDBDownloadDirectoryPath(Context context) {
         return (context.getExternalFilesDir(null) + File.separator) + "OpenCellID/";
     }
+
+    /**
+     * The interface to be implemented by the caller of RequestTask so it can perform contextual
+     * actions once the async task is completed.
+     *
+     * E.g. rechecking current cell in the newly updated database after OCID download.
+     */
+    public interface AsyncTaskCompleteListener {
+        void onAsyncTaskSucceeded(String result);
```

Comment:
If you would only ever call this when the result is "Successful", there's no need for `String result`.